@navikt/core/react/src/util/i18n/locales/en.ts

```diff
@@ -0,0 +1,27 @@
+import type nb from "./nb";
+
+export default {
+  FileUpload: {
+    dropzone: {
+      button: "Select file",
```

Comment:
Would "Pick" be more correct here?
```suggestion
      button: "Pick file",
```

Reply:
Same for "buttonMultiple" if so

Reply:
I was actually planning to check what browsers use for the native file input before pushing, but I forgot.

It looks like pick/select/choose are almost interchangeable: https://ell.stackexchange.com/questions/126487/pick-vs-select-vs-choose
As mentioned on that page, I think pick and select are better suited for when the options are presented to the user, as in a date picker or a radio group.

I think we should have similar text as browsers use for the native file input. It seems that most browsers have "Choose File", except Firefox which has "Browse". When it comes to casing, I think we should avoid UpperCamelCase b.c. I think it's mostly an American thing (?), but also b.c. I wouldn't know when to use it and not 😅

That's why I suggest "Choose file".
@JulianNymark What do you think?

Reply:
(just some screengrabs from the MDN input type=file)
Edge / Chrome:
<img width="240" alt="image" src="https://github.com/user-attachments/assets/1bb47190-bbcc-4e13-9fa3-27745072434c">
FF:
![image](https://github.com/user-attachments/assets/04c876ed-bc74-47b8-b2b6-65327ea1a14c)

I think `Choose` works well (given there's no consensus out there, I'll just _choose_ the one I like the best 😂 )

Reply:
For the casing, I don't think it makes a huge difference, for a button It might seem appropriate to use title casing (when each word is capitalized except for the "plumbing" words like `for`, `of`, `and` etc). Since it then feels a bit more emphasized and distinct from prose tekst. Since buttons are usually supposed to be short and emphasized, but I'm also not really sure about conventions for casing on more specific features of the web 😅. It feels like Title case makes sense though.